standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveAlterHandler.java

```diff
@@ -209,7 +214,13 @@ public void alterTable(RawStore msdb, Warehouse wh, String catName, String dbnam
         boolean tableInSpecifiedLoc = !oldtRelativePath.equalsIgnoreCase(name)
```

Comment:
these are not required if replDataLocationChanged is true

Reply:
OK